framework/log/FileTarget.php

```diff
@@ -117,9 +120,21 @@ public function export()
             $this->rotateFiles();
             @flock($fp, LOCK_UN);
             @fclose($fp);
-            @file_put_contents($this->logFile, $text, FILE_APPEND | LOCK_EX);
+            $writeResult = @file_put_contents($this->logFile, $text, FILE_APPEND | LOCK_EX);
+            if ($writeResult === false) {
+                throw new LogRuntimeException('Unable to export log through file!');
```

Comment:
We have to be careful about it since errors stack is quite fragile and there could be totally unrelated errors. Here I've suggested it because of silencing operator in `@file_put_contents`. It may fail for too many reasons and not knowing exact one would cause debug hell for sure.